Default location of wgetrc is in /usr/local/etc. However, we configure
wget with option --sysconfdir=/etc, so the default location of wgetrc is
in /etc. sample.wgetrc, sample.wgetrc.munged_for_texi_inclusion and 
wget.texi shall be updated with this change. As this is Solaris specific
patch it will not be pushed upstream.

--- wget-1.21.4/doc/sample.wgetrc
+++ wget-1.21.4/doc/sample.wgetrc
@@ -10,7 +10,7 @@ ##   $ info wget.info 'Startup File'
 ## Or online here:
 ##   https://www.gnu.org/software/wget/manual/wget.html#Startup-File
 ##
-## Wget initialization file can reside in /usr/local/etc/wgetrc
+## Wget initialization file can reside in /etc/wgetrc
 ## (global, for all users) or $HOME/.wgetrc (for a single user).
 ##
 ## To use the settings in this file, you will have to uncomment them,
@@ -22,7 +22,7 @@ ## For example ftp_proxy, ftp-proxy and
 
 
 ##
-## Global settings (useful for setting up in /usr/local/etc/wgetrc).
+## Global settings (useful for setting up in /etc/wgetrc).
 ## Think well before you change them, since they may reduce wget's
 ## functionality, and make it behave contrary to the documentation:
 ##
--- wget-1.21.4/doc/sample.wgetrc.munged_for_texi_inclusion
+++ wget-1.21.4/doc/sample.wgetrc.munged_for_texi_inclusion
@@ -10,7 +10,7 @@ ##   $ info wget.info 'Startup File'
 ## Or online here:
 ##   https://www.gnu.org/software/wget/manual/wget.html#Startup-File
 ##
-## Wget initialization file can reside in /usr/local/etc/wgetrc
+## Wget initialization file can reside in /etc/wgetrc
 ## (global, for all users) or $HOME/.wgetrc (for a single user).
 ##
 ## To use the settings in this file, you will have to uncomment them,
@@ -22,7 +22,7 @@ ## For example ftp_proxy, ftp-proxy and
 
 
 ##
-## Global settings (useful for setting up in /usr/local/etc/wgetrc).
+## Global settings (useful for setting up in /etc/wgetrc).
 ## Think well before you change them, since they may reduce wget's
 ## functionality, and make it behave contrary to the documentation:
 ##
--- wget-1.21.4/doc/wget.texi
+++ wget-1.21.4/doc/wget.texi
@@ -195,7 +195,7 @@ @item
 Most of the features are fully configurable, either through command line
 options, or via the initialization file @file{.wgetrc} (@pxref{Startup
 File}).  Wget allows you to define @dfn{global} startup files
-(@file{/usr/local/etc/wgetrc} by default) for site settings. You can also
+(@file{/etc/wgetrc} by default) for site settings. You can also
 specify the location of a startup file with the --config option.
 To disable the reading of config files, use --no-config.
 If both --config and --no-config are given, --no-config is ignored.
@@ -204,7 +204,7 @@ 
 @ignore
 @c man begin FILES
 @table @samp
-@item /usr/local/etc/wgetrc
+@item /etc/wgetrc
 Default location of the @dfn{global} startup file.
 
 @item .wgetrc
@@ -3185,8 +3185,7 @@ @cindex wgetrc location
 @cindex location of wgetrc
 
 When initializing, Wget will look for a @dfn{global} startup file,
-@file{/usr/local/etc/wgetrc} by default (or some prefix other than
-@file{/usr/local}, if Wget was not installed there) and read commands
+@file{/etc/wgetrc} by default and read commands
 from there, if it exists.
 
 Then it will look for the user's file.  If the environmental variable
@@ -3197,7 +3196,7 @@ If @code{WGETRC} is not set, Wget will t
 
 The fact that user's settings are loaded after the system-wide ones
 means that in case of collision user's wgetrc @emph{overrides} the
-system-wide wgetrc (in @file{/usr/local/etc/wgetrc} by default).
+system-wide wgetrc (in @file{/etc/wgetrc} by default).
 Fascist admins, away!
 
 @node Wgetrc Syntax, Wgetrc Commands, Wgetrc Location, Startup File
